ArrayPartition.java

```diff
@@ -0,0 +1,29 @@
+// Time Complexity : O(n log n)
+// Space Complexity : O(1)
```

Comment:
SC is O(log n ) to O(n) due to sorting

Reply:
Sure, thanks for pointing it out. Will make the change.